src/shared/format-table.c

```diff
@@ -2522,6 +2544,14 @@ int table_print(Table *t, FILE *f) {
                         fputc('\n', f);
                         n_subline ++;
                 } while (more_sublines);
+
+                if (typesafe_bsearch(&i, t->delimiter_map, t->n_delimiter_map, row_compare)) {
+                        for (size_t j = 0; j < display_columns; j++)
+                                for (size_t w = 0; w < width[j]; w++)
+                                        fputs(special_glyph(SPECIAL_GLYPH_LINE_HORIZONTAL), f);
+
+                        fputc('\n', f);
+                }
```

Comment:
so the lines separating the unit types in the "systemctl" output are currently drawn via ANSI underline stuff. should we do the same here?

there should probably be only one way how we draw horizontal lines in tables i guess.

Reply:
I find the ANSI underline stuff rather ugly. I'd much rather we switch to using this new thing everywhere.